problem-based-packs/insecure-transport/go-stdlib/http-request.yaml

```diff
@@ -14,6 +14,10 @@ rules:
           http.$FUNC("=~/[hH][tT][tT][pP]://127.0.0.1.*/", ...)
       - pattern-not:
           client.$FUNC("=~/[hH][tT][tT][pP]://127.0.0.1.*/", ...)
+      - pattern-not:
+          http.$FUNC("=~/[hH][tT][tT][pP]://localhost.*/", ...)
```

Comment:
crafted subdomains can bypass this check (and the `127.0.0.1` check):

https://127.0.0.1.requestcatcher.com/
https://localhost.requestcatcher.com/

Probably not important here as users aren't *trying* to evade this regex but maybe a future improvement.

Reply:
that's a good point -- hopefully users aren't trying to evade this, but we never truly know 🤔